tests/lsan/suppressions.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@asomsiko how the file should be used/consumed?

Reply:
added a comment